pkg/extensions/main.go

```diff
@@ -5,8 +5,10 @@ import (
 	// remove the cron (v1) dependency
 
 	_ "cloud.google.com/go/kms/apiv1"
+	_ "github.com/Azure/azure-sdk-for-go/sdk/keyvault/azkeys/crypto"
 	_ "github.com/Azure/azure-sdk-for-go/services/keyvault/v7.1/keyvault"
 	_ "github.com/Azure/go-autorest/autorest"
+	_ "github.com/Azure/go-autorest/autorest/adal"
```

Comment:
I added `+/adal` as well because apparently [that one](https://github.com/Azure/go-autorest/blob/master/autorest/adal/go.mod) and [its parent module](https://github.com/Azure/go-autorest/blob/master/autorest/go.mod) are different Go modules.

That does make sense, doesn't it?

Reply:
It seems so but when/how is that package used?

Reply:
Tricky one because it's part of the Enterprise code base -- [here](https://github.com/grafana/grafana-enterprise/blob/main/src/pkg/extensions/kmsproviders/azurekv/provider.go#L12).

That's why it's added at this `main.go` within `pkg/extensions` and as an underscore import.